src/ophys_etl/transforms/classification.py

```diff
@@ -302,6 +301,36 @@ def downsample(trace: np.ndarray, input_fps: int, output_fps: int):
     return downsample
 
 
+def load_model(classifier_model_uri: str) -> Any:
+    """Load a classifier model given a valid URI.
+
+    Parameters
+    ----------
+    classifier_model_uri : str
+        A valid URI that points to either an AWS S3 resource or a
+        local filepath. URI validity is only guaranteed by the
+        'InferenceInputSchema'.
+
+    Returns
+    -------
+    Any
+        A loaded ROI classifier model.
+    """
+    if classifier_model_uri.startswith("s3://"):
+        s3 = boto3.client("s3")
+        parsed = urlparse(classifier_model_uri, allow_fragments=False)
+
+        with tempfile.TemporaryFile() as fp:
+            s3.download_fileobj(Bucket=parsed.netloc,
+                                Key=parsed.path.lstrip("/"),
+                                Fileobj=fp)
+            fp.seek(0)
+            model = joblib.load(fp)
```

Comment:
Not specific to this PR, but, we can download and deserialize any file out there, which is a bit of a security risk. Any thoughts? We could restrict to a set of listed buckets, with an explicit override required for anything else. I'm not sure `hmac` is getting at my concern. I doubt someone will get into one of our braintv buckets and inject something, although, ... possible. More, if another user points this at some random file on the net...

Reply:
Would restricting it however make it less reusable if the external user has their own classifier saved in some other bucket?